omniport/core/public/index.html

```diff
@@ -30,6 +30,10 @@
   <link rel="stylesheet" type="text/css" href="/static/semantic-ui/semantic.min.css" />
   <link rel="stylesheet" type="text/css" href="/static/semantic-ui/semantic.fix.css" />
 
+  <link rel="stylesheet" type="text/css" href="/static/full-calendar-semantic-ui/daygrid.main.css" />
+  <link rel="stylesheet" type="text/css" href="/static/full-calendar-semantic-ui/core.main.css" />
+  <link rel="stylesheet" type="text/css" href="/static/full-calendar-semantic-ui/semantic.fullCalendar.css" />
```

Comment:
So you need to install `@fullcalendar` packages from `npm` **_and_** `link` to stylesheets?

Reply:
@algomaster99 @SuyashSalampuria

Reply:
Maybe we could get around this. @SuyashSalampuria just found a resource that could help.